sites/public/pages/eligibility/householdSize.tsx

```diff
@@ -71,22 +65,24 @@ const EligibilityBedrooms = () => {
           }}
         />
         <div className="form-card__lead pb-0 pt-8">
-          <h2 className="form-card__title is-borderless">{t("eligibility.bedrooms.prompt")}</h2>
+          <h2 className="form-card__title is-borderless">
+            {t("eligibility.householdSize.prompt")}
+          </h2>
         </div>
         <Form onSubmit={handleSubmit(onSubmit)}>
           <div className="form-card__group">
-            <fieldset>
-              <legend className="sr-only">{t("eligibility.bedrooms.prompt")}</legend>
-              <FieldGroup
-                type="checkbox"
-                name="bedrooms"
-                fields={bedroomsOptions}
-                error={errors.bedrooms != null}
-                errorMessage={t("errors.selectAtLeastOne")}
-                validation={{ required: true }}
-                register={register}
-              />
-            </fieldset>
+            <legend className="sr-only">{t("eligibility.householdSize.prompt")}</legend>
+            <Select
+              id="householdSize"
+              name="householdSize"
+              label={t("eligibility.householdSize.srCountLabel")}
+              describedBy="householdSize-description"
+              validation={{ required: true }}
+              register={register}
+              controlClassName="control"
+              options={householdSizeRanges}
```

Comment:
Can this just be the numeric range? To alleviate the need to have translations for the numbers?

Reply:
Actually you're right - the mocks have "How many people will be living here?" as the question and "1 person" ... as the options. That SGTM (or this if you've already talked to @tomgebauer).

Reply:
updated